packages/frontend/src/components/MkEmojiPicker.vue

```diff
@@ -470,6 +472,15 @@ defineExpose({
 						min-width: 0;
 					}
 				}
+
+				> .body.reactionPickerExpandWide {
+					display: flex;
+					flex-wrap: wrap;
+
+					> .item {
+						aspect-ratio: unset;
```

Comment:
Maybe it would be better flip the flag (`reactionPickerSquare = !reactionPickerExpandWide` or something?) so that we don't need to use `unset`.

```css
> .body {
  display: flex;
  /* ... */

  &.reactionPickerSquare {
    display: grid;
    /* ... */
    > .item {
      aspect-ratio: 1 / 1;
      /* ... */
    }
  }
}
```